packages/internal/src/tokens.js

```diff
@@ -0,0 +1,35 @@
+// @ts-check
+
+/** @typedef {'IST' | 'BLD'} TokenKeyword */
+
+/**
+ * This is defined by ERTP. For dependency pragmatism it's repeated here. We
+ * rely on the static type check and unit tests to detect any incompatibility.
```

Comment:
git doesn't recognize this as a rename. It looks like only this comment changed. I assume the rest stayed the same.